tmt/convert.py

```diff
@@ -922,7 +922,12 @@ def read_polarion_case(
     if polarion_case.tags:
         if not current_data.get('tag'):
             current_data['tag'] = []
-        for tag in polarion_case.tags.split():
+        separator = ' '
```

Comment:
A comment would be nice, something along the lines of "the space is the default, but let's try to autodetect a separator, maybe we could do better". The "why" is not clear from this sequence of tests.